tool/data/export.go

```diff
@@ -25,3 +25,11 @@ func ListAvailableRules() ([]string, error) {
 	}
 	return ruleFiles, nil
 }
+
+func ReadEmbedFile(path string) ([]byte, error) {
+	bs, err := dataFs.ReadFile(path)
```

Comment:
I think that this can be simplified to 
```suggestion
	return dataFs.ReadFile(path)
```
as we check error after `ReadEmbedFile` call